README.md

```diff
@@ -353,6 +349,25 @@ export class SettingsComponent {
 }
 ```
 
+### Set and get Initial Value
```

Comment:
I think we can make it short and clear. Just write the two functions without the whole explanation around it.

Reply:
juste like this : ?

### `setInitialValue(name, value)` - Set Initial Value

```ts
formsManager.setInitialValue('form', initialValue);
```
### `setInitialValue(name)` - Get Initial Value

```ts
formsManager.getInitialValue('form');
```
If the initial value was not defined, the get function will return undefined.

Reply:
Yes